docs/configuration/configuration.md

```diff
@@ -61,6 +61,10 @@ global:
   # How frequently to evaluate rules.
   [ evaluation_interval: <duration> | default = 1m ]
 
+  # File to which PromQL queries are logged.
+  # Reloading the configuration will reopen the file (e.g. after rotation).
```

Comment:
I'd exclude the bit in brackets.

Also put this last, it's very different to the other settings as it's not semantic.

Reply:
> Per the dev summit this should include IP addresses when it's coming from the query API.

Does the engine know about it yet?

Reply:
It doesn't, as there's a separation of concerns.